projectcard/examples/transit-route_addition.yml

```diff
@@ -0,0 +1,45 @@
+project: New Green Transit
+tags:
+   - new transit
+transit_route_addition:
+  routes:
+    - route_id: abc
+      route_long_name: green_line
+      route_short_name: green
+      route_type: 3
+      agency_id: The Bus
+      trips:
+        - direction_id: 0
+          headway_secs:
+            ('6:00','12:00'): 600
+            ('12:00','13:00'): 900
```

Comment:
headway_secs is a list
        - ('6:00', '10:00'): 600
        - ('10:00', '15:00'): 900

Reply:
gotcha. will fix.

Reply:
Just FYI (meant to say this before) - json says you can't have a tuple as an object key, so this will be stored as a string technically in json and then evaluated.

Reply:
yeah it works. transit_route_addition will [process the time range tuple as str ](https://github.com/wsp-sag/network_wrangler/blob/selection-refactor-metcouncil-test/network_wrangler/transit/projects/add_route.py#L149)